cmd/lakectl/cmd/log.go

```diff
@@ -149,6 +155,15 @@ var logCmd = &cobra.Command{
 	},
 }
 
+func MustSliceNonEmptyString(list string, v []string) []string {
```

Comment:
Also - can be in common_helpers

Reply:
Replaced it with simple slice.Contains check.
Thinking maybe to change the implementation to trim each item and drop empty string instead of reporting an error. 
It will be better implementation - but I don't want to do two things in the same PR.